docs/guides/getting-started/getting-started-scaling-and-tuning.adoc

```diff
@@ -0,0 +1,64 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/links.adoc" as links>
+
+<@tmpl.guide
+title="Scaling"
+summary="Get started with {project_name} scaling and tuning">
+
+Once you have {project_name} started you can next consider adapting your instance to whatever load may be needed. Scaling and tuning can help you address:
+
+- minimizing resource utilization
```

Comment:
My review commit adds a blank line here. In AsciiDoc, those blank lines preceding a list are necessary, as otherwise no list is rendered, but a regular text block.

Reply:
Thanks for catching that. The default eclipse AsciiDoc preview rendering for some reason is lenient here - I'll switch to a better plugin.